jerry-core/jrt/jrt.h

```diff
@@ -120,7 +120,11 @@ extern void __noreturn jerry_unimplemented (const char *, const char *, const ch
 #define JERRY_DDDLOG(...) JERRY_DLOG (__VA_ARGS__)
 #endif /* !JERRY_ENABLE_LOG */
 
+#if defined (__TARGET_ESP8266)
+#define JERRY_ERROR_MSG(...) printf (__VA_ARGS__)
```

Comment:
I think we should keep core sources as much target independent as possible, please implement `fprintf` in ESP8266 sources.